### PR TITLE
Rebrand to tamagotchi 

### PR DESCRIPTION
**Pull request by iSasuke7**
If you take care of it, it will grow into a healthy pet!

Remember to feed him or he will die!

### CODE REVIEW

File: EXPLANATIONS.md [hunk 1]
```diff
@@ -28,12 +28,12 @@ Invalidation time
 
 Related to DateUpdated, this can be thought of as a cache invalidation time.
 When the dumper is run, it checks for if an entity (say, a User) has changed
-since the last export. If there has been a change, the new User will always
-be saved. However, if the User is the same as the last export, there is a
-problem. If the exporter only saves on updates, there is an information gap
-between User updates, even though in each individual export the exporter knew
+since the last tamagotchi. If there has been a change, the new User will always
```
Comment (expectocode) [inline EXPLANATIONS.md]: Did you mean to write 'the last egg' here? Seems a little inconsistent

File: telegram_export/__init__.py [hunk 2]
```diff
@@ -1,2 +0,0 @@
-"""Components for telegram-export"""
```
Comment (expectocode) [inline telegram_export/__init__.py]: Forgot to add something? Tut tut.

**Comment by expectocode** [comment]
I do wonder if the copyright holders would appreciate the April Fools joke, but I suspect not.

File: EXPLANATIONS.md (hunk 1 above)
Comment (Lonami) [reply EXPLANATIONS.md]: Have you really read the pull request thoroughly?

File: README.rst [hunk 3]
```diff
@@ -1,4 +1,4 @@
-telegram-export
+tamagotchi
 ===============
```
Comment (Lonami) [inline README.rst]: You missed to shorten this.

File: tamagotchi/__main__.py [hunk 4]
```diff
@@ -13,11 +13,11 @@
 
 import tqdm
 import appdirs
-from telethon import TelegramClient, utils
+from garry import TelegramClient, utils
```
Comment (Lonami) [inline tamagotchi/__main__.py]: You even took into consideration the fact that you renamed `telethon` → `garry`… Nice touch.

File: EXPLANATIONS.md (hunk 1 above)
Comment (expectocode) [reply EXPLANATIONS.md]: Yeah, I was bored.

**Pull request closed without merging**